server/views/layout/default.njk

```diff
@@ -67,6 +67,13 @@
     {% component 'header', { navLinks: navLinks } %}
 
     <div id="main" class="main" role="main">
+      <div class="row bg-yellow {{ {s:3} | spacingClasses({padding: ['top', 'bottom']}) }} {{ {s:'HNL5'} | fontClasses }}">
+        <div class="container">
+          <p class="no-margin">
+          Wellcome Collection will close at 18:00 on Thursday and Friday this week so that our visitors and colleagues are able to travel home more safely. If you have any questions or concerns, please contact us on <a href="mailto:info@wellcomecollection.org">email</a> or by phone +44 (0)20 7611 2222.
```

Comment:
Is it OK to 'hide' the email address here? I think elsewhere we took the decision to spell it out in full.